athena-cloudwatch-metrics/src/main/java/com/amazonaws/athena/connectors/cloudwatch/metrics/MetricsRecordHandler.java

```diff
@@ -90,7 +93,7 @@
     private static final Table METRIC_DATA_TABLE = new MetricSamplesTable();
 
     //Used to handle throttling events by applying AIMD congestion control
-    private final ThrottlingInvoker invoker = ThrottlingInvoker.newDefaultBuilder(EXCEPTION_FILTER).build();
+    private final ThrottlingInvoker invoker = ThrottlingInvoker.newDefaultBuilder(EXCEPTION_FILTER).withInitialDelayMs(140).withIncrease(20).build();
```

Comment:
I'd recommend creating constants for these two throttling constants with a comment about how they were derived.

Reply:
done